test/game/game_setup_test.exs

```diff
@@ -19,28 +20,68 @@ defmodule GameSetupTest do
         %Players{player_one: %Player{type: HumanPlayer, symbol: "X"}, player_two: %Player{type: HumanPlayer, symbol: "O"}}
     end
 
-    test "returns players containing a Human player 1 and Computer player 2 with their assosciated symbols" do
-      user_inputs = ["1", "2"]
+    test "returns players containing a Human player 1 and an Easy Computer player 2 with their assosciated symbols" do
+      user_inputs = ["1", "2", "1"]
       start_supervised!({@mock_command_line.communicator, user_inputs})
 
       assert GameSetup.setup_players(@mock_command_line) ==
         %Players{player_one: %Player{type: HumanPlayer, symbol: "X"}, player_two: %Player{type: EasyComputerPlayer, symbol: "O"}}
     end
 
-    test "returns players containing a Computer player 1 and Human player 2 with their assosciated symbols" do
-      user_inputs = ["2", "1"]
+    test "returns players containing a Human player 1 and an Unbeatable Computer player 2 with their assosciated symbols" do
+      user_inputs = ["1", "2", "2"]
+      start_supervised!({@mock_command_line.communicator, user_inputs})
+
+      assert GameSetup.setup_players(@mock_command_line) ==
+        %Players{player_one: %Player{type: HumanPlayer, symbol: "X"}, player_two: %Player{type: HardComputerPlayer, symbol: "O"}}
+    end
+
+    test "returns players containing an Easy Computer player 1 and Human player 2 with their assosciated symbols" do
```

Comment:
```suggestion
    test "returns players containing an Easy Computer player 1 and Human player 2 with their associated symbols" do
```

Typo

Reply:
Oh no I made alot of those 😂  my bad Ill fix it now!